charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+---
+eol-exporter:
+  enabled: true
+  namespaceOverride: ''
+  applicationName: eol-exporter
+  externalSecrets:
+    enabled: false
+  deployment:
+    enabled: true
+    image:
+      repository: saritasallc/eol-exporter
+      tag: prod-b9828d4
+      digest: ''  # if set to a non empty value, digest takes precedence on the tag
+      pullPolicy: IfNotPresent
+    initContainers: []
+    command: ''
+    args: []
+    envFrom: {}
+    env:
+      ENVIRONMENT:
+        value: prod
+      CONFIG_YAML_PATH:
+        value: config.yaml
+      JOB_INTERVAL_HOURS:
+        value: '24'
```

Comment:
also add 
![image](https://github.com/user-attachments/assets/2f3040d7-345d-4aee-a3c7-ebe17a6cc0cd)
as you also expect it via env var

Reply:
Done